plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/DBUtils.java

```diff
@@ -1702,17 +1702,19 @@ public static DBSObject getActiveInstanceObject(@NotNull DBCExecutionContext exe
     }
 
     @Nullable
-    public static DBSObject getSelectedObject(@NotNull DBCExecutionContext context)
-    {
-        DBCExecutionContextDefaults contextDefaults = context.getContextDefaults();
-        if (contextDefaults != null) {
-            DBSSchema defaultSchema = contextDefaults.getDefaultSchema();
-            if (defaultSchema != null) {
-                return defaultSchema;
-            }
-            DBSCatalog defaultCatalog = contextDefaults.getDefaultCatalog();
-            if (defaultCatalog != null) {
-                return defaultCatalog;
+    public static DBSObject getSelectedObject(DBCExecutionContext context)
```

Comment:
Where did the annotation go? If it is nullable indeed, wouldn't it be better to locate the source of `null` instead of fixing the receiver?